master/pkg/archive/archive.go

```diff
@@ -80,6 +94,17 @@ func RootItem(path string, content []byte, mode int, fileType byte) Item {
 
 // UserItem returns a new Item which will be owned by the user under which the container runs.
 func UserItem(path string, content []byte, mode int, fileType byte, userID int, groupID int) Item {
+	if fileType == tar.TypeSymlink {
```

Comment:
just for consistency, can you make `RootItem` above behave the same? or, maybe better, remove "Linkname" from "Item" and do the mapping with the rest of the logic in "func tarArchive"

Reply:
Will address on dispatcher branch where I can test it, before merging.